### ISSUE TITLE
Pdoc version number

### ISSUE THREAD

**Issue by redchillipadi** [dracwyrm/gentoo-ebuilds#12]
Just had a thought about the verson bump we did for pdoc to 0.3.2, should it be called something like 0.3.1-r1 instead? [Upstream](https://github.com/BurntSushi/pdoc) are using the sequence 0.3.0, 0.3.1, so things might get messy when they release 0.3.2 and we have already used that name.

**Comment by redchillipadi** [comment]
Its nice to see that the packages are gradually getting added into the tree. Opencollada is in, and I'm not sure what happened to opencv, I must have blinked and it is gone too!

**Comment by dracwyrm** [comment]
I'm glad they finally pushed it to the tree as well. It's a newer version so it benefits LibreOffice as well. :)

This is the official 0.3.2 release: https://github.com/BurntSushi/pdoc/releases :P

**Comment by redchillipadi** [comment]
Oh, right! Well we can close this issue then :)

**Comment by dracwyrm** [comment]
Oh. Forgot to say. I was working on OpenCV for Amynka. :) (S)He is a Gentoo dev. I recently did a lot of hacking with python on the OpenVDB ebuild and helped her with that. I had to rewrite the python detection routine. It was hard coded to detect version 2 and 3 of python, but if both were detected, v2 was used. This made it impossible to do Single_python_targets, as if you wanted 3.x and had 2.x installed, you would get 2.x instead. I had to fix that part. I'm now an expert of tricky python installations. haha.
